numba_dpex/core/kernel_interface/indexers.py

```diff
@@ -219,6 +219,12 @@ def __repr__(self):
         """
         return self.__str__()
 
+    def __eq__(self, other):
```

Comment:
why do we have to overload `NdRange` and not overloading `Range`?

Reply:
`Range` is a sub-type of `tuple` and that is the reason `__eq__` works out of the box.